README.md

```diff
@@ -171,23 +171,9 @@ avoid using `eyre::Report` as your public error type.
 
 ## No-std support
 
-**NOTE**: tests are currently broken for `no_std` so I cannot guarantee that
-everything works still. I'm waiting for upstream fixes to be merged rather than
-fixing them myself, so bear with me.
-
-In no_std mode, the same API is almost all available and works the same way. To
-depend on Eyre in no_std mode, disable our default enabled "std" feature in
-Cargo.toml. A global allocator is required.
-
-```toml
-[dependencies]
-eyre = { version = "0.6", default-features = false }
-```
-
-Since the `?`-based error conversions would normally rely on the
-`std::error::Error` trait which is only available through std, no_std mode will
-require an explicit `.map_err(Report::msg)` when working with a non-Eyre error
-type inside a function that returns Eyre's error type.
+No-std support was removed in 2020 in [commit 608a16a] due to unaddressed upstream breakages.
+[commit 608a16a]:
+https://github.com/eyre-rs/eyre/pull/29/commits/608a16aa2c2c27eca6c88001cc94c6973c18f1d5
```

Comment:
:+1: seems good - if it's not too much work it would be nice to record what needs to be changed upstream to add support back

Reply:
I'll ask jane what exactly broke it next time we talk, but I figure it might have shifted over the last three years =]

Reply:
I honestly don't recall, I think it was mostly error in core or the lack there of, and the horrible hacks that we had to work around that.